staff/mgarciarojas/Javascript exercises/isVowel.js

```diff
@@ -0,0 +1,10 @@
+
+//isVowel() function
+//Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+
+
+function isVowel(letter){
+    return ['a','e','i','o','u'].includes(letter)
```

Comment:
Perfect! Now as a challenge, try to do this w/ Regular Expressions → https://apuntes-javascript-intermedio.netlify.com/expresiones_regulares/

Reply:
Done